lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -46,13 +46,7 @@
         </button>
     <% } else {%>
         <div class="no-action-message">
-            <%- gettext('Coming Soon') %>
-        </div>
-        <div class="enrollment-opens">
-            <%- gettext('Enrollment Opens on') %>
-            <span class="enrollment-open-date">
-                <%- enrollment_open_date %>
-            </span>
+            <%- gettext('Not Currently Available') %>
```

Comment:
It's not my understanding that we want to completely remove the "Coming Soon" message. We only want to show "Not Currently Available" if there are no future course runs available. However, we want to show "Coming Soon" if there are future course runs available, but the enrollment isn't open yet. Can you confirm with Lauren on what is the requirement here?